oga/frontend/src/containers/Answer/NewAnswer.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import './NewAnswer.css';
+
+import { connect } from 'react-redux';
+import * as actionCreators from '../../store/actions/';
+
+import { Redirect } from 'react-router-dom';
+import { push } from 'connected-react-router';
+import Map from '../Map/GoogleMap';
+import PushNotification from '../../components/PushNotification/PushNotification'
+import NewQuestion from '../QuestionList/NewQuestion/NewQuestion';
+
+class NewAnswer extends Component {
+  state = {
+    answer_content: '....',
+  }
+
+  componentDidMount() {
+    this.props.onGetQuestion(this.props.match.params.id);
+  }
+
+  postQuestionHandler = () => {
+    if (this.state.answer_content !== '' && this.props.selectedQuestion.content)
```

Comment:
I don't understand this condition. Can you please explain? If we got the question from backend, shouldn't if have content?

Reply:
First I changed it to 
postAnswerHandler = () => {
    if (this.state.answered && this.props.selectedQuestion.content)

it means when the user clicked on one of the answer which is specific to a question and the user normally came to answer page by clicking on reply button on main page, and clicked submit button on this page, then that answer will be posted. I meant this.

Reply:
the selected_question will be set by using id from url.

Reply:
and Question have question_type whose default is "LINE"